sdk/tools/log2lines/cache.c

```diff
@@ -18,8 +18,10 @@
 
 #include "log2lines.h"
 
-static char *cache_name;
-static char *tmp_name;
+static char CacheName[PATH_MAX];
+static char *cache_name = CacheName;
```

Comment:
Why not using directly `static char cache_name[PATH_MAX];` ? (and same remark for `tmp_name` below).

Reply:
oh OK the code changes the pointer.